src/azul/project/hca/indexer.py

```diff
@@ -11,6 +11,14 @@ class Indexer(BaseIndexer):
     def mapping(self) -> JSON:
         return {
             "dynamic_templates": [
+                {
+                    "donor_age_range": {
+                        "path_match": "contents.donors.organism_age_range_in_seconds",
+                        "mapping": {
+                            "type": "double_range"
```

Comment:
Can you document the reasoning behind our choice of `double` instead of `float` in a comment here?

Reply:
The "float with 2 decimal precision" phrase is misleading and irrelevant. Please change to:

> A `float` (single precision IEEE-754) can represent all integers up to 16,777,216. If we used `float` values for organism ages in seconds, we would not be able to accurately represent an organism age of 16,777,217 seconds. That is 194 days and 15617 seconds. A double precision IEEE-754 representation loses accuracy at 9,007,199,254,740,993 which is more than 285616415 years.
>
> Note that Python's `float` uses double precision IEEE-754. 
>
> (https://docs.python.org/3/tutorial/floatingpoint.html#representation-error)